StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/Vertical Saved Payment Method Screen/CheckmarkCircleView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  CheckmarkCircleView.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/9/24.
+//
+
+import Foundation
+import UIKit
+
+/// Draws a circle with the desired fill color with a white checkmark in the center
+final class CheckmarkCircleView: UIView {
```

Comment:
Omitted snapshot tests for this class and relying on the `VerticalSavedPaymentOptionsViewControllerSnapshotTests`. Can add them if we feel they will be useful.